apps/web/app/(pages)/page.tsx

```diff
@@ -5,30 +5,6 @@ import LandingSubmission from "@good-dog/components/LandingSubmission";
 import ProjectGallery from "@good-dog/components/ProjectGallery";
 
 export default function Home() {
-  const Splash = () => {
```

Comment:
I missed this in my review where this was added, but we shouldn't be defining components inside other components like this.